src/classes/RD2_ValidationService.cls

```diff
@@ -382,9 +410,19 @@ public with sharing class RD2_ValidationService {
         }
     }
 
+    private void validateElevateRDScheduleCannotBeInTheFuture(npe03__Recurring_Donation__c rd, ErrorRecord errorCollection) {
```

Comment:
I was able to create a new Elevate RD with a Start Date in the future, but then I couldn't update it without changing the date to Today, is that an issue?

Reply:
That's the expected behavior. We are preventing the future effective date for Elevate **update** only because Elevate does not support 2 schedules and NPSP RD does. The field effective date on edit means when will the new schedule start.